frontend/plugin-manager/OBSPluginManager.cpp

```diff
@@ -0,0 +1,41 @@
+#include "ObsPluginManager.hpp"
+
+#include <OBSApp.hpp>
+#include <QVBoxLayout>
+#include <QHBoxLayout>
+#include <QCheckBox>
+#include <QScrollArea>
+#include <QDialogButtonBox>
+#include <QLabel>
+
+#include "moc_OBSPluginManager.cpp"
+OBSPluginManager::OBSPluginManager(std::vector<OBSModuleInfo> const& modules, QWidget* parent)
+	: QDialog(parent), _modules(modules), ui(new Ui::OBSPluginManager)
+{
+	setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+
+	ui->setupUi(this);
+
+	std::sort(_modules.begin(), _modules.end(), [](const OBSModuleInfo& a, const OBSModuleInfo& b) {
+		auto aName = a.display_name != "" ? a.display_name : a.module_name;
```

Comment:
Is `display_name` a `std::string`? If so, the `empty()` class method will return true also for empty strings (and not just uninitialised strings).